ops/terraform/variables.tf

```diff
@@ -1,12 +1,16 @@
 variable "client_id" {}
 variable "name" {}
+variable "postgres_server_id" {}
```

Comment:
I removed it for now.  I added it because I added the `azurerm_private_endpoint`, which can be an optional architectural design if we want a higher level security.  For now, and to unblock the engineers, it is best to remove it.